packages/coinstac-api-server/src/data/resolvers.js

```diff
@@ -444,12 +444,18 @@ const resolvers = {
           ) {
             return Boom.forbidden('Action not permitted');
           } else {
-            return rethink.table('pipelines').get(args.pipelineId)
+            return new Promise.all([
+              connection,
+              rethink.table('pipelines').get(args.pipelineId)
               .delete({ returnChanges: true })
-              .run(connection).then(res => connection.close().then(() => res))
+              .run(connection),
+              rethink.table('consortia').filter({ pipelineId: args.pipelineId })
```

Comment:
Consortia table does not have a field called `pipelineId`. There's actually one called `activePipelineId`. Also there's a field called `pipelines` which is an array of id's of pipelines, which may also need to be updated.

Reply:
@rochaeb 
I updated field name to `activePipelineId`.

By the way,  I cannot see `pipelines` field in Consortia table.
This is current Consortia table structure.
`
{
  "activeComputationInputs": [],
  "activePipelineId": "",
  "description": "ACON",
  "id": "0fd110df-4528-466b-bbf1-b1cf856daae9",
  "members": [
    "test1"
  ],
  "name": "ACon",
  "owners": [
    "test1"
  ],
  "tags": []
}`

Reply:
Updating the `activePipelineId` field to empty string works (checked on my side)
But I updated codebase to remove `activePipelineId` key when the active pipeline is deleted from consortia to get rid of your worries, @rochaeb